src/passport.js

```diff
@@ -0,0 +1,62 @@
+const passwordGenerator = require('generate-password');
+const bcrypt = require('bcrypt');
+const genUsername = require('unique-username-generator');
+const { sendEmail } = require('./services/mail');
+const User = require('./models/user');
+
+const saltRounds = 10;
+
+const googleConfigs = {
+  clientID: process.env.GAPP_CLIENT_ID,
+  clientSecret: process.env.GAPP_CLIENT_SECRET,
+  callbackURL: 'http://localhost:3000/api/auth/google/callback',
```

Comment:
I forget to mention this😅 , but instead of writing `http://localhost:3000` ,use the server base url in .env file  so that we change the variable there instead of doing it here . Same happened with me and Sherrya notified me about this